### PR TITLE
Align Margins on Register Test Screen (EXPOSUREAPP-7348)

### PR DESCRIPTION
**Pull request by axelherbstreith**
Aligned the cards spacing between the cards to match the figma design

### CODE REVIEW

File: Corona-Warn-App/src/main/res/layout/fragment_submission_dispatcher.xml [hunk 1]
```diff
@@ -107,7 +107,7 @@
                     layout="@layout/include_dispatcher_card"
                     android:layout_width="0dp"
                     android:layout_height="wrap_content"
-                    android:layout_marginTop="@dimen/spacing_normal"
+                    android:layout_marginTop="16dp"
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/res/layout/fragment_submission_dispatcher.xml:110]: In the design there should not be any margins for in the images it is aligned to bottom right corner
Screenshot 2021-05-19 at 13 01 06

Comment (axelherbstreith) [reply Corona-Warn-App/src/main/res/layout/fragment_submission_dispatcher.xml:110]: is now fixed

**Review by mtwalli — DISMISSED** [review]
Tested , Looks good

**Review by harambasicluka — CHANGES_REQUESTED** [review]
image
the illustrations on the card are overlapping, therefore the border radius is gone O.o

**Comment by mtwalli** [comment]
> image
> the illustrations on the card are overlapping, therefore the border radius is gone O.o

Good catch!, @axelherbstreith  I think we can make the parent layout of the items a `MaterialCard` and it will crop the content  with the rounded corners

**Comment by axelherbstreith** [comment]
@mtwalli I fixed it now with changing the `ImageView` to `ShapeableImageView` and add a corner radius to the bottom right that fits the corder or the parent

**Review by mtwalli — APPROVED** [review]

**Review by harambasicluka — APPROVED** [review]

**Pull request merged**